src/components/calculator.js

```diff
@@ -1,44 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
+import btnValues from './btnNames';
+import '../App.css';
 
-class Calculator extends React.Component {
```

Comment:
Please fix the design issue as it was pointed out by the previous code reviewer.
![design](https://user-images.githubusercontent.com/35504149/149529979-280962d2-ecb3-498a-902e-c1c4af938819.png)